package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "next lint",
     "lint:fix": "next lint --fix",
     "prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --ignore-unknown",
-    "prettier:check": "npm run prettier -l",
+    "prettier:check": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --check --ignore-unknown",
```

Comment:
`npm run prettier --check` doesn't work?

Reply:
This will run for all the files, then we will have to run fix for all the files, thus will throw error.

Reply:
I tried npm run prettier --check doesn't work, it doesn't work.